tutorial.md

```diff
@@ -116,8 +118,11 @@ sudo apt-get update && apt-get install -y libglib2.0-dev \
                                           libseccomp-dev \
                                           libgpgme11-dev \
                                           libdevmapper-dev \
-                                          make \
-                                          git
+					  libudev-dev \
+					  go-md2man \
```

Comment:
Fair point, at some point during the process I got an error that it was not installed. It might be because I was switching between the root / non root user. I've removed the line.